pkgs/development/python-modules/black/default.nix

```diff
@@ -54,6 +54,8 @@ buildPythonPackage rec {
   ] ++ lib.optionals stdenv.isDarwin [
     # fails on darwin
     "test_expression_diff"
+    "test_bpo_2142_workaround"
```

Comment:
```suggestion
    # Fails on some darwin systems and hydra
    # see https://github.com/NixOS/nixpkgs/pull/130785
    "test_bpo_2142_workaround"
```

Reply:
Apologies for forgetting all about this.